src/Symfony/Component/Serializer/Encoder/ChainDecoder.php

```diff
@@ -35,16 +35,17 @@ public function __construct(array $decoders = array())
      */
     final public function decode($data, $format, array $context = array())
     {
-        return $this->getDecoder($format)->decode($data, $format, $context);
+        return $this->getDecoder($format, $context)->decode($data, $format, $context);
     }
 
     /**
      * {@inheritdoc}
      */
-    public function supportsDecoding($format)
+    public function supportsDecoding($format/*, array $context = array()*/)
```

Comment:
It's is very unlikely that some extended this class (I never seen this at least).